### PR TITLE
Added information for Kevin Eaton

### PR DESCRIPTION
**Pull request by kevineaton**

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -1,8 +1,3 @@
-#### Name: [Darsh Naik](https://github.com/DarshNaik)
```
Comment (StevenCopeland) [inline CONTRIBUTORS.md]: Why did you delete this name? Please add it back to this file.

**Comment by Roshanjossey** [comment]
Hi  @kevineaton, I'm working on a similar project to help beginners to get started with contributing to open source projects.

Do check it out https://github.com/Roshanjossey/first-contributions
